osi_sensorview.proto

```diff
@@ -29,33 +48,94 @@ message SensorView
+    // which are used in the detected features.
+    //
+    optional Identifier sensor_id = 3;
 
-    // Lidar-Specific SensorView
+    // The virtual mounting position of the sensor (origin and orientation
+    // of the sensor coordinate system) given in vehicle coordinates [1].
+    // The virtual position pertains to the sensor as a whole, regardless
+    // of the actual position of individual physical detectors, and governs
+    // the sensor-relative coordinates in detected objects of the sensor
+    // as a whole.  Individual features detected by individual physical
+    // detectors are governed by the actual physical mounting positions
+    // of the detectors, as indicated in the technology-specific sub-views
+    // and sub-view configurations.
+    //
+    // \arg \b x-direction of sensor coordinate system: sensor viewing direction
+    // \arg \b z-direction of sensor coordinate system: sensor (up)
+    // \arg \b y-direction of sensor coordinate system: perpendicular to x and z right hand system
     //
-    optional LidarSensorView lidar_sensor_view = 1001;
+    // \par References:
+    // [1] DIN ISO 8855:2013-11
+    //
+    // \note This field is usually static during the simulation.
+    // \note The origin of vehicle's coordinate system in world frame is
+    // ( \c Vehicle::base.position + Inverse_Rotation_yaw_pitch_roll(\c Vehicle::base.orientation) * \c Vehicle::bbcenter_to_rear) .
+    // The orientation of the vehicle's coordinate system is equal to the orientation
+    // of the vehicle's bounding box \c Vehicle::base.orientation.
+    //
+    optional MountingPosition mounting_position = 4;
 
-    // Camera-Specific SensorView
+    // The root mean squared error of the mounting position.
     //
-    optional CameraSensorView camera_sensor_view = 1002;
+    optional MountingPosition mounting_position_rmse = 5;
 
-    // Ultrasonic-Specific SensorView
+    // Host Vehicle Data
     //
-    optional UltrasonicSensorView ultrasonic_sensor_view = 1003;
+    // Host vehicle data is data that the host vehicle knows about itself,
+    // e.g. from location sensors, internal sensors and ECU bus data, etc.,
+    // that is made available to sensors as input.
+    //
+    optional HostVehicleData host_vehicle_data = 6;
+
+    // Ground truth w.r.t. global coordinate system
+    //
+    // This is the ground truth that is provided to the sensor model by the
+    // simulation environment. It is filtered as per the requirements of the
+    // sensor model as expressed by the SensorViewConfiguration message(s)
+    // that where exchanged during the simulation initialization phase.
+    //
+    // \note The host vehicle is always contained in the ground truth provided,
+    // regardless of any filtering.
+    //
+    optional GroundTruth global_ground_truth = 7;
+
+    // The ID of the host vehicle in the ground_truth data.
+    //
+    optional Identifier host_vehicle_id = 8;

```

Comment:
also HostVehicleData?

Reply:
I seperated host_vehicle_data and host_vehicle_id since they really have nothing in common: host_vehicle_data is proper new input, that is not duplicated in global_ground_truth, whereas host_vehicle_id really is more or less rendundant data that might really be included in ground_truth (and to some degree is since global_ground_truth already contains the host_vehicle flag).

Reply:
see comment above -  the information on the perfect host vehicle position is in the vehicles including the flag "host vehicle"